packages/api/cms-api/src/blocks/blocks-meta.service.ts

```diff
@@ -1,9 +1,15 @@
 import { getBlocksMeta } from "@comet/blocks-api";
 import { OnModuleInit } from "@nestjs/common";
 import { promises as fs } from "fs";
+import { format, resolveConfig } from "prettier";
```

Comment:
Prettier is a dev dependency, so this wouldn't work in production, right? Wouldn't it be a better solution to do this in a precommit hook with lint-staged?

Reply:
Also, block-meta.json in `@comet/cms-api` isn't formatted

Reply:
It think it's even valid to ignore the block-meta.json file for Prettier

Reply:
Agree.